src/GuiHandler.java

```diff
@@ -0,0 +1,373 @@
+    	if(txtTryAWord.getText().matches(".*\\d.*")) {errorPopupHandler(0, "", "Het woord mag alleen letters bevatten"); return;}
+    	if(answer.length() <= 0) {errorPopupHandler(0, "", "Voer een woord in om te raden"); return;}
+
+    	String word = txtTryAWord.getText();
+        int inputLenght = word.length();
+        
+        try 
+        {
+        	if(inputLenght < 3) throw new IllegalArgumentException();
+        	
+        	if(!word.equalsIgnoreCase(answer)) {loseGameHandler(""); return;}
+        	winGameHandler();
+        }
+        catch(IllegalArgumentException ilex)
+        {
+        	errorPopupHandler(inputLenght, "Voer minimaal 1 woord in van 3 letters", "Voer minimaal 1 woord in van 3 letters");
+        }
+    }
+    
+    //Helper functions
+    private void correctGuesHandler(String letter)
+    {
+    	StringBuilder fillCorrectLetter = new StringBuilder(correctLetters);
+    	char correctCharLetter = letter.charAt(0);
+    	
+    	long amountCharInString = answer.length() - (correctLetters.chars().filter(ch -> ch != correctCharLetter).count());
+    	int correctLetterIndex = answer.indexOf(letter);
+
+    	if(correctLetters.indexOf(letter) >= 0)
+    	{
+    		
+    		for(int i = 0; i < amountCharInString + 1; i++)
+        	{
+        		if(i == 0)
+        		{
+        			correctLetterIndex = answer.indexOf(letter);
+        		}
+        		else if(i > 0)
+        		{
+        			correctLetterIndex = answer.indexOf(letter, answer.indexOf(letter) + i);
+        		}
+        	}
+    	}
+
+    	if(correctLetterIndex < 0) {wrongGuesHandler(letter); return;}
+    	fillCorrectLetter.setCharAt(correctLetterIndex, correctCharLetter);
+    	correctLetters = fillCorrectLetter.toString();
+    	
+    	lblLettersCorrect.setText("Letters correct: " + correctLetters);
+		txtTryALetter.clear();
+    }
+    
+    private void wrongGuesHandler(String letter)
+    {
+		mistakes++;
+		if(mistakes >= 11) {loseGameHandler("lastImg"); return;}
+		
+    	String currentWrongLetters = lblLettersWrong.getText();
+		lblLettersWrong.setText(currentWrongLetters + " " + letter);
+		
+		imgHangman.setImage( new Image( getClass().getResource("img/hangman_" + mistakes +".jpg").toString()));
+		
+		txtTryALetter.clear();
+    }
+    
+    private void errorPopupHandler(int inputLenght, String tekst1, String tekst2)
+    {
+    	Alert alert = new Alert(AlertType.ERROR);
+    	alert.setTitle("ERROR");
+    	
+    	if(inputLenght == 0)
+    	{
+    		alert.setContentText(tekst2);
+    		alert.showAndWait();
+    		return;
+    	}
+    	
+    	if(inputLenght == 8)
+    	{
+    		alert.setContentText(tekst1);
+    		alert.showAndWait();
+    		return;
+    	} 
+    	else if(inputLenght < 3)
+    	{
+    		alert.setContentText(tekst1);
+    		alert.showAndWait();
+    		return;
+    	}
+    	
+    	if(inputLenght > 1)
+    	{
+    		alert.setContentText(tekst1);
+    	}
+    	else
+    	{
+    		alert.setContentText(tekst2);
+    	}
```

Comment:
Kun je hier niet ook een error string van maken zodat je alle fouten in een oogwink kan zien?